packages/tools/devtools/devtools-view/src/test/OpLatencyView.test.tsx

```diff
@@ -21,7 +22,13 @@ import { OpLatencyView } from "../components";
 
 describe("OpLatencyView component tests", () => {
 	it("Renders as expected", async (): Promise<void> => {
-		render(<OpLatencyView />);
+		render(
+			<MessageRelayContext.Provider
+				value={new WindowMessageRelay("fluid-framework-devtools-inline")}
```

Comment:
We have a MockMessageRelay (example usage [here](https://github.com/microsoft/FluidFramework/blob/d5b79831bd267e88419ad21ae3a487ead3bd018f/packages/tools/devtools/devtools-view/src/test/VisualTree.test.tsx#L54-L75)) that I think might be a better fit for tests. For this test in particular we don't even need it to have any handlers, I think.

Reply:
I think we do need it because the OpLatencyView uses it and expects the context to already be initialized. I'll switch to using the mock.